vhdbuilder/packer/components.json

```diff
@@ -418,7 +418,7 @@
       ]
     },
     {
-      "fileName": "azure-vnet-cni-linux-amd64-v*.tgz",
```

Comment:
errr why change this? bad ctrl-f replace?

Reply:
When we loop through the files looking to verify what we have defined here in components we've already deleted the .tgz. So it'll error out. But it will successfully find this ref as a folder 

https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/test/linux-vhd-content-test.sh#L37